packages/quizzes-dashboard/development.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:11-alpine
+
+USER node
+VOLUME /quizzes/dashboard
```

Comment:
I don't think we should mark the source code as a volume since it won't contain any data that we want to persist?